### PR TITLE
feat: configurable pg connection lifetime and idle timeouts

### PR DESCRIPTION
**Pull request by rafaelcr**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -27,12 +31,13 @@ PG_APPLICATION_NAME=stacks-blockchain-api
 # PG_PRIMARY_DATABASE=
 # PG_PRIMARY_SCHEMA=
 # PG_PRIMARY_SSL=
+# PG_PRIMARY_IDLE_TIMEOUT=
+# PG_PRIMARY_MAX_LIFETIME=
```
Comment (CharlieC3) [inline .env:35]: Are these being used in code anywhere?

Comment (rafaelcr) [reply .env:35]: Yep, they will be chosen if they're available: https://github.com/hirosystems/stacks-blockchain-api/blob/0eeee9558ae6ad4d0d233dca49c87430c0da57a6/src/datastore/connection.ts#L125-L144

Comment (CharlieC3) [reply .env:35]: Interesting,  thanks!

**Review by CharlieC3 — APPROVED** [review]
:shipit:

**Pull request merged**